server/Makefile

```diff
@@ -0,0 +1,15 @@
+CC=${CROSS_COMPILE}gcc
+
+all: default
+
+default: aesdsocket
+
+writer: aesdsocket.c
+	$(CC) -c -o aesdsocket.o aesdsocket.c
```

Comment:
@SundarKrishnakumar consider -Wall and -Werror here to help catch mistakes at the compile stage